.travis.yml

```diff
@@ -1,24 +1,2 @@
 language: java
```

Comment:
Always do the simplest thing that could possibly work. All that other stuff is for fancier builds that need to run on different versions of Java, which we don't need.